README.md

```diff
@@ -116,7 +116,7 @@ dependencies {
 
 The most bleeding-edge version is: [![Bleeding-edge](https://api.bintray.com/packages/yahoo/maven/fili/images/download.svg)](https://bintray.com/yahoo/maven/fili/_latestVersion)
 
-The most recent stable version is: [![Stable](https://img.shields.io/badge/Stable-0.6.29-blue.svg)](https://bintray.com/yahoo/maven/fili/0.6.29)
+The most recent stable version is: [![Stable](https://img.shields.io/badge/Stable-0.7.36-blue.svg)](https://bintray.com/yahoo/maven/fili/0.7.36)
```

Comment:
Tucking the stable version ID down here, rather than at the top, is very much intentional. We'd _rather_ users are on the bleeding-edge, since that lets us get feedback quickly, but we also want a path for those who are more cautious and want to absorb fewer changes.

Also, having 2 version numbers / download links in the top would be a bit confusing as well 😄